event_handler.py

```
@@ -837,6 +841,90 @@ async def handle_group_del_robot(self, event_data: Dict[str, Any]) -> bool:
+        member_name = raw_username or stats_manager.get_user_display_id(member_openid) or member_openid[:12]
+        message_archive.log_system_message(
+            group_openid=group_openid,
+            content="加入了群聊",
+            user_id=member_openid,
+            username=member_name,
+            event_type="GROUP_MEMBER_ADD",
+        )
+
+        # 发送欢迎消息
+        welcome_msg = f"欢迎新人 {member_name} 入群~ 🎉"
+        asyncio.create_task(self._send_group_welcome_farewell(group_openid, welcome_msg))
```

Ruff (0.15.20)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)